scripts/deeplink-dispatcher.sh

```
@@ -0,0 +1,316 @@
+    return 0
+  fi
+}
+
+# Function to send link to selected iOS Simulators
+send_to_ios() {
+  local link="$1"
+  shift
+  local simulators=("$@")
+  
+  for sim in "${simulators[@]}"; do
+    local sim_name=$(echo "$sim" | cut -d':' -f1)
```

Shellcheck

[warning] Declare and assign separately to avoid masking return values.

(SC2155)